configure.ac

```diff
@@ -0,0 +1,57 @@
+AC_INIT(FlexDLL,0.35,alain@frisch.fr,flexdll,http://alain.frisch.fr/flexdll.html)
+#AC_COPYRIGHT(Copyright 2016 ?)
+
+AC_ARG_VAR([MINGW_PREFIX],[Prefix for the mingw-w64 i686 cross-compiler])
+AC_ARG_VAR([MINGW64_PREFIX],[Prefix for the mingw-w64 x64 cross-compiler])
+AS_IF([test x"${MINGW_PREFIX}" = x""],[MINGW_PREFIX=i686-w64-mingw32])
+AS_IF([test x"${MINGW64_PREFIX}" = x""],[MINGW64_PREFIX=x86_64-w64-mingw32])
```

Comment:
I have used `AC_ARG_VAR` as it allows the prefix for the mingw compilers to be overridden. I've done that only because they are written to `version.ml`, but I'm not sure why/when you'd want to override this anymore (was it support the older mingw compilers too?).